src/BuildingBlocks/EventBus/EventBusServiceBus/EventBusServiceBus.cs

```diff
@@ -38,14 +43,14 @@ public void Publish(IntegrationEvent @event)
             var jsonMessage = JsonSerializer.Serialize(@event);
```

Comment:
This line and the line below can be removed, since the constructor for BinaryData will serialize to JSON using System.Text.Json by default.